apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.ISessionPool.cs

```diff
@@ -27,6 +27,7 @@ internal interface ISessionPool
         {
             SpannerClient Client { get; }
             IClock Clock { get; }
+            Task<PooledSession> WithFreshTransactionOrNewAsync(PooledSession session, TransactionOptions transactionOptions, CancellationToken cancellationToken);
```

Comment:
I'm glad ISessionPool is internal, otherwise this would have been a breaking change :)